ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/LimitPayloadSize.java

```diff
@@ -25,15 +32,21 @@ public static LimitPayloadSize toMB(int mb) {
   }
 
   @Override
-  protected PubsubMessage processElement(PubsubMessage message) throws Exception {
-    message = PubsubConstraints.ensureNonNull(message);
-    int numBytes = message.getPayload().length;
-    if (numBytes > maxBytes) {
-      countPayloadTooLarge.inc();
-      throw new PayloadTooLargeException("Message payload is " + numBytes + "bytes, larger than the"
-          + " configured limit of " + maxBytes);
-    }
-    return message;
+  public Result<PCollection<PubsubMessage>, PubsubMessage> expand(
```

Comment:
Basically, for every class that currently extends `MapElementsWithErrors` the logic from `processElements` can be moved into `expands`.